src/networkfilemanager.cpp

```diff
@@ -1622,7 +1629,19 @@ CurlFileHandle::CurlFileHandle(PJ_CONTEXT *ctx, const char *url, CURL *handle)
 #if defined(SSL_OPTIONS)
     // https://curl.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
     auto ssl_options = static_cast<long>(SSL_OPTIONS);
+#if CURL_AT_LEAST_VERSION(7, 71, 0)
+    if (pj_context_get_native_ca(ctx)) {
+        ssl_options = ssl_options | CURLSSLOPT_NATIVE_CA;
+    }
+#endif
     CHECK_RET(ctx, curl_easy_setopt(handle, CURLOPT_SSL_OPTIONS, ssl_options));
+#else
+#if CURL_AT_LEAST_VERSION(7, 71, 0)
```

Comment:
It looks to me that this branch ( SSL_OPTIONS not defined but curl >= 7.71) cannot be taken as SSL_OPTIONS is anterior to 7.71

Reply:
As far as I understand, SSL_OPTIONS is a preprocessor definition that is specific to PROJ, it is not defined by curl. It was introduced in this pull request: #3936

Reply:
ah indeed, I missed that. Looks ok then